src/main/scala/scalaz/stream/process1.scala

```diff
@@ -656,6 +656,20 @@ object process1 {
   def sum[N](implicit N: Numeric[N]): Process1[N,N] =
     fold(N.zero)(N.plus)
 
+  /**
+   * Emits all elements of the input except the first one.
+   *
+   * @example {{{
+   * scala> Process(1, 2, 3).tail.toList
+   * res0: List[Int] = List(2, 3)
+   *
+   * scala> Process[Int]().tail.toList
+   * res1: List[Int] = List()
+   * }}}
+   */
+  def tail[I]: Process1[I, I] =
+    receive1(_ => id)
+
```

Comment:
ah, ok I think since ++ is lazy, it probably won't have that much overhead. But ok LGTM.